apps/darts-modernisation/stg/base/darts-api-function.enc.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 data:
-  DARTS_API_DB_CONNECTION_STRING: ENC[AES256_GCM,data:YIK4EY0SqJZbV2YWFvnADZzgd+nUKsPX3mvlNVzAM5oUL1qiDzxWuNR+SCOQQmAySguRvDOuagLpvhHYf6AZpgxru0tLFJ8IQ1Pxy081gEaA5+fdzlEfZpdO9uaJrAv/RVElevAahE7HlePuF1D/xZP95FQrHLq3VFy6qA==,iv:16okVrHh5ewjqyn+oNIHrYNHIc9tqAxuvgauoIa46pY=,tag:8E03qjOE9JRZmFiYffW+xw==,type:str]
+  DARTS_API_DB_CONNECTION_STRING: darts-api-test.postgres.database.azure.com
```

Comment:
This connection string value is encrypted with SOPS because it contains the database password. It's value should something like 

```
postgres://pgadmin:XXXPASSWORDXXX@darts-api-test.postgres.database.azure.com:5432/darts
```

Reply:
have restored that value and encrypted the test file